config/env/production.js

```diff
@@ -78,6 +78,13 @@ module.exports = {
     callbackURL: '/api/auth/paypal/callback',
     sandbox: false
   },
+  owasp: {
+      allowPassphrases       : true,
+      maxLength              : 128,
+      minLength              : 10,
+      minPhraseLength        : 20,
+      minOptionalTestsToPass : 4,
+  },
```

Comment:
No one should touch `default.js` anyway, it's wrooong and should cause tests to fail. ;-) They will modify `local-{env}.json` files instead.